src/components/ProgressIndicator/ProgressIndicatorDisplay.js

```diff
@@ -173,6 +175,18 @@ const ProgressIndicatorDisplay = ({
               isCurrent={step.isCurrent}
             />
           ))}
+          {showAppointment && (
+            <ProgressItem completed={isAppointment}>
+              <LinkOrSpan
+                to={'/appointment'}
+                useLink={areApplicableStepsCompleted}
+                isActive={isSummary}
```

Comment:
we probably need to refactor the progress indicator component more, since the "appointment" step checkmark is now tied to the summary page which is too late.

It would maybe make sense to use a context of relevant items/steps here and have that state managed by the parent container so that this component is a bit more general-purpose.

Reply:
Should we make it in the separate PR? With some regression testing?  For now I wanted to add the appointment flow with minimal changes to the existing components. For now only `ProgressIndicatorDisplay` is reused, but the change is quite small.

Reply:
yeah, we have some existing planned refactoring for the PI component anyway since the markup is... sketchy :grimacing: good to leave it until later.